### PR TITLE
feat(BBD-655): Add support for default isActive behaviour

### PR DESCRIPTION
**Pull request by Vic-Dev**

### CODE REVIEW

File: src/navigation-display/index.ts [hunk 1]
```diff
@@ -89,6 +87,7 @@ namespace NavigationDisplay {
     value: string;
     display: Display;
     label: string;
+    defaultIsActive: boolean;
```
Comment (effervescentia) [inline src/navigation-display/index.ts]: we can just call this `active`

File: src/navigation/index.ts [hunk 2]
```diff
@@ -23,10 +24,11 @@ class Navigation {
 
   updateFields = (navigations: Store.Indexed<Store.Navigation>) =>
     this.set({
-      fields: navigations.allIds.map((value) => ({
+      fields: navigations.allIds.map((value, index) => ({
         value,
         display: this.props.display[value],
         label: this.props.labels[value],
+        defaultIsActive: index < this.props.defaultOpen
```
Comment (effervescentia) [inline src/navigation/index.ts]: `active`

File: src/navigation/index.ts [hunk 3]
```diff
@@ -11,7 +11,8 @@ class Navigation {
   props: Navigation.Props = {
     display: {},
     labels: {},
-    collapse: true
+    collapse: true,
+    defaultOpen: 3
```
Comment (effervescentia) [inline src/navigation/index.ts]: default should be to have them all open
also rename to `uncollapsed`? thoughts?

Comment (Vic-Dev) [inline src/navigation/index.ts]: Maybe have it name "isActive" and 3 sets the first 3 active on fields to true, true sets all the active on fields to true, false sets all active on fields to false?

**Pull request merged**